vagrant/provisioning/roles/minio/tasks/main.yml

```diff
@@ -3,19 +3,18 @@
   become: yes
   group:
     name: minio
-    gid: 9002
     state: present
 - name: create minio user
   become: yes
   user:
     name: minio
     group: minio
     shell: /sbin/nologin
-    home: /opt/minio
+    home: /home/minio
 - name: create minio directories
   become: yes
   file:
-    path: /opt/minio/{{item}}
+    path: "{{root_folder}}/minio/{{item}}"
```

Comment:
We want a folder structure like so:

{{ root_folder }}/app/minio

{{ root_folder }}/log/minio

{{ root_folder }}/data/minio

{{ root_folder }}/tmp/minio

{{ root_folder }}/install/minio